web/client-api/src/main/java/io/deephaven/web/client/api/WorkerConnection.java

```diff
@@ -478,11 +482,11 @@ this, ConfigService.GetConfigurationConstants, new ConfigurationConstantsRequest
                     }
 
                     // Read the timeout from the server, we'll refresh at less than that
-                    result.getMessage().getConfigValuesMap().forEach((item, key) -> {
-                        if (key.equals("http.session.durationMs")) {
-                            sessionTimeoutMs = Double.parseDouble(item.getStringValue());
-                        }
-                    });
+                    constants = result.getMessage();
```

Comment:
The config fetch is cheap (no locks, no updategraph or tables to touch, just directly querying the Configuration object), and simpler than the Handshake call we did before (which actually required two http calls when using http2, thanks browsers). Flight auth v2 effectively requires that the client have a "no op" http call it can make, this is the same one that the C++, Python, and Java (and R? not Go i know) clients use, so JS is now more consistent with them.

This changed in 0.36 in anticipation of https://github.com/deephaven/deephaven-core/pull/6032.